USAGE.md

```diff
@@ -0,0 +1,104 @@
+# Amazon Alexa Smart Home Skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This page describes how to use the skill certified by Amazon.
+This skill allows you to connect your openHAB setup through the [myopenHAB.org](http://myopenHAB.org) cloud service to Amazon Alexa.
+(See [README.md](https://github.com/openhab/openhab-alexa/blob/master/README.md) for other setup options and development information.)
+
+With this skill you can control items that respond to on/off and numeric commands.
+The Skill as well has limited support for thermostats.
+The skill is supported for English (U.S.), English (U.K.), and German languages. 
+
+## General Configuration Instructions
+
+### Requirements
+
+* [openHAB Cloud Connector](http://docs.openhab.org/addons/io/openhabcloud/readme.html) configured using myopenHAB.org
+* Amazon account
+* Amazon Echo, Amazon Echo Dot or compatible Alexa speaker or solution (e.g. [Reverb](https://reverb.ai/))
+
+### 1. Item Configuration
+
+You have to tag your items to make them useable by Alexa.
+See [Tagging](http://docs.openhab.org/configuration/items.html#tagging) for more details.
+
+* **Items via .items - File**
+
+  Please make sure to place your tag infront of the channel-informations. e.g.
+  See [Item Definition and Syntax](http://docs.openhab.org/configuration/items.html#item-definition-and-syntax)
+  
+   ```java
+   Switch Kitchen_Light "Kitchen Light" <light> (gKitchen) [ "Lighting" ] {channel="..."}
+   ```
+   
+  Some examples of tagged items are:
+  
+  ```java
+  Switch Kitchen_Light "Kitchen Light" <light> (gKitchen) [ "Lighting" ]
+  Dimmer Bedroom_Light "Bedroom Light" <light> (gBedroom) [ "Lighting" ]
+  Number Bedroom_Temperature "Bedroom Temperature" (gBedroom) [ "CurrentTemperature" ]
+  
+  Group gDownstairsThermostat "Downstairs Thermostat" (gFF) [ "Thermostat" ]
+  Number Downstairs_Thermostat_CurrentTemp "Downstairs Thermostat Current Temperature" (gDownstairsThermostat) [ "CurrentTemperature" ]
+  Number Downstairs_Thermostat_Target_Temperature "Downstairs Thermostat Target Temperature" (gDownstairsThermostat) [ "TargetTemperature" ]
+  String Downstairs_Thermostat_Heating_Cooling_Mode "Downstairs Thermostat Heating/Cooling Mode" (gDownstairsThermostat) [ "homekit:HeatingCooling
+  ```
+
+* **Items via PaperUI**
+ 
+   There is no easy way to tag your Items via PaperUI.
+ 
+   Some users managed to tag items via REST API.
+   Have a loot at [this post](https://community.openhab.org/t/apply-tags-to-items-added-linked-in-paper-ui/19443/11?u=mboremski).
```

Comment:
I think we should remove links to community posts for official documentation, plus that post is really not very clear on adding tags, what if you removed "**Items via PaperUI**" and instead had:

**Other configuration options**
  * The interactive REST API interface available through the openHAB 2 dashboard
  * PaperUI does not yet allow manipulation of tags

Reply:
will do